apis/cloudstack/src/main/java/org/jclouds/cloudstack/domain/UsageRecord.java

```diff
@@ -476,7 +595,7 @@ public String getIpAddress() {
       return this.ipAddress;
    }
 
-   public boolean isSourceNAT() {
+   public Boolean isSourceNAT() {
```

Comment:
Add the `@Nullable` annotation, for consistency.

Reply:
@mcdonnell-john looks like this still needs to be addressed

Reply:
@nacx Your right, I only added the annotation to the constructor parameters. When I get home I will make this change and squash again.  Sorry about that.

Reply:
Fixed that there.